apps/web/src/components/terminal/MoreMenuDropdown.tsx

```
@@ -0,0 +1,55 @@
+  return (
+    <>
+      <button
+        onClick={onToggle}
+        className={clsx(
+          'p-1 rounded',
+          'text-muted-foreground hover:text-foreground',
+          'hover:bg-card transition-colors',
+          isOpen && 'bg-card text-foreground'
+        )}
+        aria-label="More options"
+      >
```

Biome (2.3.13)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)